src/wwwroot/partials/templates/template.html

```diff
@@ -54,19 +54,45 @@ <h4 ng-show="template.id">{{'templates_information_subtitle' | translate}}{{temp
       <h2>{{'templates_edition_title' | translate}}</h2>
       <div class="flex flex--between">
         <label for="uContent">{{'templates_edition_subtitle' | translate}}</label>
+      </div>    
+      <div class="templates-htmlraw--container" ng-show="{{!forceMsEditor}}">  
+        <textarea id="uContent"
+                  name="uContent"
+                  validation-errors
+                  ng-required="{{!forceMsEditor}}"
+                  ng-model="template.content"
+                  class="input width--full"></textarea>
+      </div>
+      <div class="grid templates-selector--container" ng-show="{{forceMsEditor}}">
+        <div class="templates-mseditor--container">
+          <label>{{ 'templates_edition_mseditor_label' | translate }}</label> 
+          <a class="button-mseditor--selector" ng-click="save('mseditor');submitted = true">
```

Comment:
What is `submitted = true`?  please consider a new method in place of using _save_